views/users/edit.pug

```diff
@@ -0,0 +1,37 @@
+
+extends ../application.pug
+
+block content
+	.container
+		.row
+			.col-xs-6.offset-xs-2
+				if errors
+					each error in errors
+						div(class="alert alert-danger" role="alert")
+							span= error.message
+				br
+				h1 Update your Profile:
+				hr
+				form(action="/user/" + user.id method="POST")
+					input(type="hidden" name="_method" value="PUT")
+					div(class="form-group")
+						lable(for="profileImageURL") Picture Link:
```

Comment:
typo, should be label, also your for="" attribute should match the input id, thus user-profileImageURL. You have typos in other form elements here too